lower-bound-requirements.txt

```diff
@@ -6,8 +6,7 @@ jsonschema==3.0.0
 jsonpatch==1.15
 pyyaml==5.1
 # xmlio
-uproot3==3.14.1
-uproot==4.0.0
+uproot==4.1.0
```

Comment:
You might want to loosen that to anything in 4.1.*. They're will be a 4.1.1 to fix a performance bug soon.

Reply:
This is intentional. The `lower-bound-requirements.txt` is only used in the minimum supported dependencies workflow that we run to empirically test the lower bounds we state in `setup.cfg` and `setup.py`.

https://github.com/scikit-hep/pyhf/blob/ce2ffabaaeadf2733d4d61284d750933f34cee28/.github/workflows/lower-bound-requirements.yml#L30 

If there were bug fixes that we became dependent on this would help us catch them so we could update the minimum required version of `uproot` in the `xmlio` extra in `setup.py`. :+1: